test/e2etest.py

```diff
@@ -1237,6 +1237,12 @@ def test_post_incident(sample_user, sample_team, sample_application_name, sample
     assert re.status_code == 200
     assert re.json() == {'owner': sample_user, 'incident_id': incident_id, 'active': False}
 
+    # Invalid claim owner
+    re = requests.post(base_url + 'incidents/%d' % (incident_id, ), json={
```

Comment:
Should be able to do:

    base_url + 'incidents/%d' % incident_id

Instead of:

    base_url + 'incidents/%d' % (incident_id, )